addon/templates/components/phone-number-field.hbs

```diff
@@ -0,0 +1,8 @@
+{{#unless required}}
+  <span class="cs-component-text-field__optional">Optional</span>
+{{/unless}}
+{{phone-number-field/input
+  class="cs-component-text-field__input"
+  required=required
+}}
+<label for={{inputId}}><span>{{label}}</span></label>
```

Comment:
This template is near identical to the TextField component. The only thing that's different is the component for the actual textfield input. As soon as we add validations, error messages, etc that could be a good opportunity to refactor so that we only use one template where we can interchange the input field.